src/components/users/userDashboard/index.js

```diff
@@ -7,29 +7,35 @@ import Table from 'react-bootstrap/Table';
 import Offcanvas from 'react-bootstrap/Offcanvas';
 import toast from 'react-hot-toast';
 import { useNewUserRequest } from "../../../hooks/mutations/useNewUserRequest";
+import { useGetAllUserRequests } from "../../../hooks/mutations/useGetAllUserRequests";
 
 
 
 
 export const UserDashboard2 = () => {
-
-   
-
     const [requestData, setRequestData] = useState({
```

Comment:
You could reformat this section to look better. Keep all the `useStates` code together (line after line), then an empty line, before custom hooks (e.g usequeries), another empty line before the functions. It'll look a lot better and it would be easier to read.